corehq/ex-submodules/casexml/apps/stock/models.py

```diff
@@ -54,7 +65,7 @@ class StockTransaction(models.Model):
     # currently supported/expected: 'stockonhand', 'receipts', 'consumption'
     type = models.CharField(max_length=20)
     # e.g. 'loss', 'transfer', 'inferred'
-    subtype = TruncatingCharField(max_length=20, null=True, blank=True)
+    subtype = TruncatingTextField(max_length=40, null=True, blank=True)
```

Comment:
(i'm genuinely conflicted on my feelings on this - not looking for a particular answer)